go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/chzyer/logex v1.1.10 // indirect
 	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e
 	github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1 // indirect
+	github.com/google/go-cmp v0.5.2
```

Comment:
I wonder if it's OK to add new dependencies in terms of pprof being a part of golang distribution.

Reply:
Done.